contents/newsletter/good-taste-great-products.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Good taste makes great products
+date: 2025-03-06
+author:
+ - danilo-campos
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/418950516_1ae2b12e_a571_48f6_a979_2c89deb7a7f9_c1c4010916.jpg
+featuredImageType: full
+tags:
+  - Engineering
+  - Product
+crosspost:
+  - Product engineers
+---
+
+When we talk about how to be a good [product engineer](/blog/what-is-a-product-engineer), we often focus on how you need to ship fast, be customer obsessed, and [take extreme ownership of ideas](https://newsletter.posthog.com/p/hiring-and-managing-cracked-engineers). 
```

Comment:
```suggestion
When we talk about how to be a good [product engineer](/blog/what-is-a-product-engineer), we often focus on how you need to ship fast, be customer obsessed, and [take extreme ownership of ideas](/founders/cracked-manifesto). 
```

Minor nit I frequently forget myself, but we should link to website versions of newsletters on the website.